src/app/game/settings/account/change/password/page.tsx

```diff
@@ -1,13 +1,40 @@
+"use client"
 import Link from "next/link";
 import { ChevronLeft } from "lucide-react";
 import NavigationFooter from "@/features/game/NavigationFooter";
 import { Label } from "@/components/ui/label";
 import { Input } from "@/components/ui/input";
 import { Button } from "@/components/ui/button";
+import { changePassword } from "@/features/auth/auth";
+import { useState } from "react";
+import { useRouter } from "next/navigation";
 
 export default function GameSettingsAccountChangePassword() {
   const pageTitle = "設定";
   const pageSubTitle = "パスワード変更";
+  const [errorMessage, setErrorMessage] = useState("");
+  const [nowPassword, setPassword] = useState("");
```

Comment:
変数名が`nowPassword`なので以下のように修正提案出してきます！
```suggestion
  const [nowPassword, setNowPassword] = useState("");
```